vars/signArtifacts.groovy

```diff
@@ -24,11 +26,21 @@ void call(Map args = [:]) {
             export UNSIGNED_BUCKET=${SIGNER_CLIENT_UNSIGNED_BUCKET}
             export SIGNED_BUCKET=${SIGNER_CLIENT_SIGNED_BUCKET}
 
-            $WORKSPACE/sign.sh ${args.artifactPath} --sigtype=${args.signatureType} --component=${args.component} --type=${args.type}
+            $WORKSPACE/sign.sh ${arguments}
```

Comment:
Removed the previous modification on `run_sign.py` for circumstance of pass `None` but add the check here so that if any arguments are not specified, it wouldn't pass to the sign script.